src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const firstNumber = (a) => {
```

Comment:
```suggestion
  const getSum = (a) => {
```
function/method name should start with a verb